src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -770,6 +769,11 @@ def exec_line(
             line_result.output[LINE_NUMBER_KEY] = index
         return line_result
 
+    def _update_operation_context(self, run_id: str):
+        operation_context = OperationContext.get_instance()
+        operation_context.run_mode = operation_context.get("run_mode", None) or RunMode.Test.name
+        operation_context.update({"flow-id": self._flow_id, "root-run-id": run_id})
```

Comment:
Do we have test to cover the context values in node runs?

Reply:
Added.